src/test/java/pages/AuthenticationPage.java

```diff
@@ -0,0 +1,88 @@
+package pages;
+
+import baseEntities.BasePage;
+import core.ReadProperties;
+import io.qameta.allure.Step;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AuthenticationPage extends BasePage {
+
+    @FindBy(xpath = "//input[@name = 'email_create']")
+    private WebElement createAnAccountEmailAddressInput;
+    @FindBy(id = "SubmitCreate")
+    private WebElement createAnAccountButton;
+    @FindBy(xpath = "//input[@name = 'email']")
+    private WebElement alreadyRegisteredEmailAddressInput;
+    @FindBy(xpath = "//input[@name = 'passwd']")
+    private WebElement alreadyRegisteredPasswordInput;
+    @FindBy(id = "SubmitLogin")
+    private WebElement signInButton;
+    @FindBy(css = ".alert.alert-danger")
+    private WebElement errorText;
+
+    public AuthenticationPage(boolean openPageByURL) {
+        super(openPageByURL);
+    }
+
+    @Override
+    protected void openPage() {
+        driver.get("http://automationpractice.com/index.php?controller=authentication&back=my-account");
+    }
+
+    @Override
+    protected boolean isPageOpened() {
+        try {
+            return signInButton.isDisplayed();
+        } catch (NoSuchElementException ex) {
+            return false;
+        }
+    }
+
+    private void setCreateEmailAddress(){
+        createAnAccountEmailAddressInput.sendKeys(ReadProperties.getInstance().getEmail());
```

Comment:
here and below, it is not a good way of doing things (setting up values from a properties file), what if you will need to set a email from another source... so, in page classes such methods should receive a value from input arg, and in test or somewhere else read properties and paste them as input arg to page methods,